main.go

```diff
@@ -29,60 +22,81 @@ func dbConn() (db *sql.DB) {
 }
 
 func Index(w http.ResponseWriter, r *http.Request) {
+
+	n := Funcionario{}
+	SlcFuncionario := []Funcionario{}
+	IndxPg := IndexPage{}
+
 	db := dbConn()
-	selDB, err := db.Query("SELECT *  FROM funcionarios ")
+	resultado, err := db.Query("SELECT *  FROM funcionarios ")
 	if err != nil {
 		panic(err.Error())
 	}
 
-	n := Names{}
-	res := []Names{}
-
-	for selDB.Next() {
+	for resultado.Next() {
 		var id int
 		var name, email string
+		var salario float64
 
-		err = selDB.Scan(&id, &name, &email)
+		err = resultado.Scan(&id, &name, &email, &salario)
 		if err != nil {
 			panic(err.Error())
 		}
 
-		n.Id = id
-		n.Name = name
-		n.Email = email
+		n.id = id
+		n.name = name
+		n.email = email
+		n.salario = salario
```

Comment:
is there a way to create it rightaway instead of line by line?

Reply:
yes, now it is in better way